insonmnia/dwh/server.go

```diff
@@ -62,6 +63,19 @@ func NewDWH(ctx context.Context, cfg *Config, key *ecdsa.PrivateKey) (*DWH, erro
 		logger: log.GetLogger(ctx),
 	}
 
+	bch, err := blockchain.NewAPI(blockchain.WithConfig(w.cfg.Blockchain))
+	if err != nil {
+		cancel()
+		return nil, errors.Wrap(err, "failed to create NewAPI")
+	}
+	w.blockchain = bch
+
+	numBenchmarks, err := bch.Market().GetNumBenchmarks(ctx)
```

Comment:
looks like you even do not need const with benchmarks - you need only to properly pass it to backend